tools/rpkg/R/Result.R

```diff
@@ -32,8 +23,6 @@ duckdb_execute <- function(res) {
   }
 }
 
-#' @rdname duckdb_result
-#' @export
```

Comment:
Was this intended?

Reply:
the constructor does not need to be exported?